include/mbedtls/ctr_drbg.h

```diff
@@ -271,13 +271,17 @@ int mbedtls_ctr_drbg_write_seed_file( mbedtls_ctr_drbg_context *ctx, const char
 int mbedtls_ctr_drbg_update_seed_file( mbedtls_ctr_drbg_context *ctx, const char *path );
 #endif /* MBEDTLS_FS_IO */
 
+#if defined(MBEDTLS_SELF_TEST)
+
 /**
  * \brief               Checkup routine
  *
  * \return              0 if successful, or 1 if the test failed
  */
 int mbedtls_ctr_drbg_self_test( int verbose );
 
+#endif
```

Comment:
Style: Please add the comment for the matching `#if`. That is: `/* MBEDTLS_SELF_TEST */`